src/toolkits/coreml_export/neural_net_models_exporter.cpp

```diff
@@ -161,6 +162,13 @@ std::shared_ptr<MLModelWrapper> export_object_detector_model(
     size_t image_height, size_t num_classes, size_t num_predictions,
     flex_dict user_defined_metadata, flex_list class_labels,
     std::map<std::string, flexible_type> options) {
+  // Get the name of image input
+  std::string input_name;
+  for (auto& it : user_defined_metadata) {
+    if (it.first == "feature") {
+      input_name = it.second.get<flex_string>();
+    }
+  }
```

Comment:
Could we just require the caller to specify this, as another argument, instead of assuming that this is in the `user_defined_metadata`?